src/components/enterprise-user-subsidy/data/service.js

```diff
@@ -1,13 +1,9 @@
-import qs from 'query-string';
-import { getAuthenticatedUser, getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
+import { getAuthenticatedHttpClient } from '@edx/frontend-platform/auth';
 import { getConfig } from '@edx/frontend-platform/config';
 
-export function fetchSubscriptionLicensesForUser(subscriptionUuid) {
+export function fetchSubscriptionLicensesForUser(enterpriseUuid) {
   const config = getConfig();
-  const user = getAuthenticatedUser();
-  const { email } = user;
-  const queryParams = { search: email };
-  const url = `${config.LICENSE_MANAGER_URL}/api/v1/subscriptions/${subscriptionUuid}/license/?${qs.stringify(queryParams)}`;
+  const url = `${config.LICENSE_MANAGER_URL}/api/v1/learner-licenses?enterprise_customer_uuid=${enterpriseUuid}`;
```

Comment:
nit: missing trailing `/` at end of `/api/v1/learner-licenses/`

This will cause a 302 redirect to the URL with trailing slash when making this request.